examples/cross-account/README.md

```diff
@@ -0,0 +1,76 @@
+This is a secure example setup to support cross-account provisioning of stacks with stacker. It:
+
+1. Sets up an appropriate [AWS Config File](https://docs.aws.amazon.com/cli/latest/topic/config-vars.html) in [.aws/config] for stacker to use, with profiles for a "master", "prod" and "stage" AWS account.
+2. Configures a stacker bucket in the "master" account, with permissions that allows CloudFormation in "sub" accounts to fetch templates.
+
+## Setup
+
+### Create IAM roles
+
+First things first, we need to create some IAM roles that stacker can assume to make changes in each AWS account. This is generally a manual step after you've created a new AWS account.
+
+In each account, create a new stack using the [stacker-role.yaml](./templates/stacker-role.yaml) CloudFormation template. This will create an IAM role called `Stacker` in the target account, with a trust policy that will allow the `Stacker` role in the master account to `sts:AssumeRole` it.
+
+Once the roles have been created, update the `role_arn`'s in [.aws/config] to match. If you created them with the name `Stacker`, then you'll just need to update the AWS account id references.
```

Comment:
I think this means to match the role that was created with the previous template - maybe show how to get that with `stacker info`?  I've even automated this in a private branch I have to mess with the bats tests:

```
# cat tests/Makefile
permissions:
	./stacker.yaml.sh | stacker build -

test: permissions
	$(eval AWS_ACCESS_KEY_ID := $(shell ./stacker.yaml.sh | stacker info - 2>&1 | awk '/AccessKeyId/ {print $$3}'))
	$(eval AWS_SECRET_ACCESS_KEY := $(shell ./stacker.yaml.sh | stacker info - 2>&1 | awk '/SecretAccessKey/ {print $$3}'))
	AWS_ACCESS_KEY_ID=$(AWS_ACCESS_KEY_ID) AWS_SECRET_ACCESS_KEY=$(AWS_SECRET_ACCESS_KEY) bats .
```

Reply:
Can't really use `stacker info` at this point, since the roles haven't actually been applied to stacker yet (chicken and egg), so I just updated this with an `aws` CLI example.